client/app/telecom/telephony/alias/configuration/agents/ovhPabx/telecom-telephony-alias-configuration-agents-ovhPabx.controller.js

```diff
@@ -181,21 +216,17 @@ angular.module('managerApp').controller('TelecomTelephonyAliasConfigurationAgent
             timeout: 20,
             wrapUpTime: 0,
           }).$promise;
-      }))
-        .then(() => {
-          this.TucToast.success(this.$translate.instant('telephony_alias_configuration_agents_add_success'));
-          return this.fetchAgentsIds().then((ids) => {
-            this.agents.ids = ids;
-          });
-        })
-        .catch((err) => {
-          this.TucToast.error([this.$translate.instant('telephony_alias_configuration_agents_add_error'), _.get(err, 'data.message')].join(' '));
-          return this.$q.reject(err);
-        })
-        .finally(() => {
-          this.addAgentForm.isAdding = false;
-          this.addAgentForm.numbers = [null];
+      })).then(() => {
+        this.TucToast.success(this.$translate.instant('telephony_alias_configuration_agents_add_success'));
+        return this.fetchAgentsIds().then((ids) => {
```

Comment:
Same thing for returning promise and move `then` block below, at the same level of `$q.all`